src/Symfony/Bundle/FrameworkBundle/Tests/CacheWarmer/SerializerCacheWarmerTest.php

```diff
@@ -112,8 +160,12 @@ public function testClassAutoloadExceptionWithUnrelatedException()
             }
         }, true, true);
 
-        $warmer->warmUp('foo');
+        try {
+            $warmer->warmUp(\dirname($file), \dirname($file));
+        } finally {
```

Comment:
```suggestion
        } catch (\DomainException) {
```

@nicolas-grekas Is it what you had in mind ?

Reply:
almost: we should rethrow the exception
and keep the other call in finally
while at it, we should move the expectException calls just before the call to wamrup (outside of the try/catch)

Reply:
Will apply the changes later today 👍